monkey/infection_monkey/post_breach/pba.py

```diff
@@ -24,17 +25,20 @@ def __init__(self, name="unknown", linux_cmd="", windows_cmd=""):
         self.command = PBA.choose_command(linux_cmd, windows_cmd)
         self.name = name
 
-    def run(self):
+    def run(self, telemetry_messenger: ITelemetryMessenger):
         """
         Runs post breach action command
         """
         if self.command:
             exec_funct = self._execute_default
             result = exec_funct()
             if self.scripts_were_used_successfully(result):
-                T1064Telem(
-                    ScanStatus.USED, f"Scripts were used to execute {self.name} post breach action."
-                ).send()
+                self._telemetry_messenger.send_telemetry(
```

Comment:
I don't think there is a `self._telemetry_messenger` the way the code is currently written. But there will be if PBAs are passed a telemetry messenger at construction instead of runtime.